releases/latest/edge/bundle.yaml

```diff
@@ -228,6 +248,7 @@ applications:
     channel: latest/edge
     scale: 1
     trust: true
+    series: focal
```

Comment:
is this required?

Reply:
yes, we published this charm with two different bases (20.04 and 22.04). `juju deploy` will default to 22.04, but we want it to default to 20.04.